src/Components/AxiosConfig/axiosConfig.js

```diff
@@ -1,11 +0,0 @@
-import React from "react";
-import axios from "axios";
-
-const axiosInstance = axios.create({
-    baseURL: 'https://api.github.com',
-    
-});
-
-export const apiKey = '6614dee7bd34ca2508d77bb0886d4ab2' ;
-
-export default axiosInstance 
```

Comment:
this is not a component: 
1. no need for react here 
2. `apiKey` is not used in the project